src/index.html

```diff
@@ -7,8 +7,86 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@500&display=swap"
+    rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="logo moyo">
+      </a>
+      <nav class="navi">
+        <ul class="navigation">
+          <li>
```

Comment:
Fix classes everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
```

Reply:
Видає таку помилку: 
src/style.css
 22:1  ✖  Expected class selector ".nav__list" to be  
   selector-class-pattern  
          kebab-case       
 33:1  ✖  Expected class selector ".nav__link" to be  
   selector-class-pattern  
          kebab-case       
 43:1  ✖  Expected class selector ".nav__item" to be  
   selector-class-pattern  
          kebab-case       
 47:1  ✖  Expected class selector ".nav__link" to be  
   selector-class-pattern  
          kebab-case